api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -1353,6 +1456,8 @@ type ClusterConfiguration struct {
 	// registries, and policies to block or allow registry hostnames.
 	// When exposing OpenShift's image registry to the public, this also lets cluster
 	// admins specify the external hostname.
+	// Changing this value will trigger a rollout for all existing NodePools in the cluster.
+	// TODO(alberto): elaborate why.
```

Comment:
nit: would be nice to link a Jira ticket here so we don't lose track of this TODO.

Reply:
There is other config that should also cause a rollout, such as proxy and featuregate